tensorflow/python/ops/math_ops.py

```diff
@@ -646,6 +646,35 @@ def imag(input, name=None):
     return gen_math_ops.imag(input, Tout=input.dtype.real_dtype, name=name)
 
 
+def arg(input, name=None):
```

Comment:
Is this not the case with `imag`? I am doing this only to maintain consistency with the way other ops are defined.